src/Gateway/Connection.php

```diff
@@ -40,7 +41,7 @@ class Connection
     private Puppet $puppet;
 
     /**
-     * @param Bitwise<\Ragnarok\Fenrir\Enums\Gateway\Intents> $intents
+     * @param Bitwise<Intents> $intents
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)